index.js

```diff
@@ -171,6 +171,16 @@ $('.hide-full-schedule').click(function() {
     $('.show-full-schedule').css('display', 'flex');
 });
 
+// Set schedule tab based on the date
+let currentDate = new Date().getDate()
+if (currentDate < 20) {
+    $('#pre-event-tab').addClass('schedule-tab-active');
+} else if (currentDate == 20 ) {
+    $('#feb-20-tab').addClass('schedule-tab-active');
```

Comment:
should also remove class of the other two for when the day switches so we don't have multiple divs with `schedule-tab-active`

Reply:
Theoretically, since this is on page load this shouldn't matter?